capstone/helpers/image_processing.py

```diff
@@ -0,0 +1,111 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import skimage.io as io
+import skimage.transform as transform
+import skimage.exposure as exposure
+import glob
+import os
+
+
+def print_columns(data, cols):
+    """
+    Splits a list into sublists and prints it out in columns
+    :params data: list of data to be printed
+    :params cols: number of columns
+    :return: print the data and return none
+    """
+    num_per_col = int(np.ceil(len(data) / cols))
+    split = [data[i:i + num_per_col] for i in range(0, len(data), num_per_col)]
+    for row in zip(*split):
+        print("".join(str.ljust(i, 30) for i in row))
+
+
+def standardize(image_name, size, path):
+    """
+    Standardizes an image to a particular size and rotation
+    :params image_name: name of new image
+    :params size: standard image size
+    :params path: path to images directory
+    :return: standardized image as numpy array
+    """
+    image = io.imread(path + image_name)
+    existing_shape = image.shape
+    if existing_shape[1] > existing_shape[0]:
+        image = transform.rotate(image, 90, resize=True)
+    if image.shape != size:
+        image = transform.resize(image, size)
+    return image
+
+
+def mean_image(image_names, shape, path):
+    """
+    Calculates the average image from a list of image names
+    :params image_names: list of image names
+    :params shape: standard image shape
+    :params path: path to images directory
+    :return: average image as numpy array
+    """
+    N = len(image_names)
```

Comment:
`N` is not that much descriptive, capital letters for variable names are not that common + you're using `N` just once there so you can simply use `len(image_names)` at line 52